### ISSUE TITLE
do run_classifier.py confused

### ISSUE THREAD

**Issue by QiuJianL** [zihangdai/xlnet#134]
Hello, author. 
    I am running the run_classifier.py on the 11GB GPU to classify the IMDB classification data set. the result,
==================
I0705 15:32:22.833007 140127178974976 run_classifier.py:807] Eval result | eval_accuracy 0.5 | eval_loss 6.055393695831299 | global_step 4000 | loss 6.055393695831299 | path ./model/model.ckpt-4000 | step 4000 | 
I0705 15:32:22.833080 140127178974976 run_classifier.py:812] ==============================================================
I0705 15:32:22.833148 140127178974976 run_classifier.py:816] Best result | eval_accuracy 0.5119600296020508 | eval_loss 0.750563383102417 | global_step 0 | loss 0.750563383102417 | path ./model/model.ckpt-0 | step 0 | 
I0705 15:32:22.833271 140127178974976 run_classifier.py:423] Create new tfrecord ./output/spiece.model.len-128.dev.predict.tf_record.
==================
 I am very confused. Do you know why? My parameters 
python run_classifier.py \
    --do_train=True \
    --do_eval=True \
    --eval_all_ckpt=True \
    --task_name=imdb \
    --data_dir=${IMDB_DIR} \
    --uncased=False \
    --spiece_model_file=${LARGE_DIR}/spiece.model \
    --model_config_path=${GS_ROOT}/${LARGE_DIR}/model_config.json \
    --init_checkpoint=${GS_ROOT}/${LARGE_DIR}/xlnet_model.ckpt \
    --max_seq_length=128 \
    --train_batch_size=8 \
    --eval_batch_size=8 \
    --num_hosts=1 \
    --num_core_per_host=1 \
    --learning_rate=2e-5 \
    --train_steps=4000 \
    --warmup_steps=500 \
    --save_steps=500 \
    --iterations=500

**Comment by lukemelas** [comment]
Are you still having this issue? Can you try decreasing the learning rate?

**Comment by QiuJianL** [comment]
@lukemelas Thank you for your suggestion. I changed learning_rate=2e-5 to learning_rate=1e-6.
 The result is
I0709 16:08:40.559786 139852461307648 run_classifier.py:817] Best result | eval_accuracy 0.7373999953269958 | eval_loss 1.2282277345657349 | global_step 2000 | loss 1.2282277345657349 | path ./model/model.ckpt-2000 | step 2000 |
